### ISSUE TITLE
organize our CSS

### ISSUE THREAD

**Issue by jiemakel** [humanitiesplusdesign/fibra#23]
Our main.styl file is cluttered, and probably contains definitions no longer in use. We should organize our CSS stylesheets into separate files depending on scope (whole app or individual component) and/or by purpose. 

I've started this by moving some of my definitions out of main.styl into sources.styl. Please do the same with your definitions, and add `// explanatory comments` to those global definitions still actually in use.

**Comment by esjewett** [comment]
Would we be open to organizing JS/Typescript along these lines as well? The current folder is getting a bit unwieldy just in terms of how many files are in there.

**Comment by jiemakel** [comment]
Yes. I thought about that too, but wasn't sure some of my magic wouldn't break if I went ahead and did it :).

I'll do the code organization once I have a bit more time to also fix stuff if it breaks.
